src/test/analyzer_test.ts

```diff
@@ -84,6 +85,32 @@ suite('Analyzer', () => {
           ['MyNamespace.SubBehavior', 'MyNamespace.SimpleBehavior']);
     });
 
+    // TODO(fks) 09-28-2016: Fix off-by-one error in warning sourceRange line
+    // numbers.
+    test('throws when cant find behavior', async() => {
+      try {
+        await analyzer.analyze('static/html-missing-behaviors.html');
+      } catch (err) {
+        assert.instanceOf(err, WarningCarryingException);
+        const warning: Warning = err.warning;
+        assert.equal(
+            err.message,
+            'Unable to resolve behavior `Polymer.ExpectedMissingBehavior`. ' +
+                'Did you import it? Is it annotated with @polymerBehavior?');
+        assert.deepEqual(warning.sourceRange, {
+          file: 'static/html-missing-behaviors.html',
+          start: {
+            line: 23,  // Actual Expected: 24
```

Comment:
Our line & position numbers are currently expected to be index-zero. This is weird in the warning use case, since most text editors are index-one.

Started an issue to discuss here: https://github.com/Polymer/polymer-analyzer/issues/332